kiali-server/values.yaml

```diff
@@ -39,6 +39,8 @@ deployment:
   # For more control over what the Kial Service Account can see, use the Kiali Operator.
   cluster_wide_access: true
   configmap_annotations: {}
+  # custom environment variables
```

Comment:
@andrewrothstein please delete this comment. It is superfluous and just adds more clutter to the list of settings here. This is the only thing that needs to change in this PR. Once this is done and I get the operator PR reviewed, this PR and the operator PR can be merged.
```suggestion
```